### PR TITLE
Changes maintainer nomination criteria to reflect reality

### PR DESCRIPTION
**Pull request by ForestEckhardt**
- Makes steering team CODEOWNERS of the governance document

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -41,17 +41,17 @@ New contributors may be self-nominated or nominated by existing contributors, an
 
 
 #### Maintainers
-Maintainers are in charge of the day to day maintenance of the team's projects. 
+Maintainers are in charge of the day to day maintenance of the team's projects.
 
 They review, approve, and merge PRs and may vote on RFCS that affect their subteams, ensuring contributions align with project goals and meet the project's quality standards.
 
-New maintainers must already be contributors, must be nominated by an existing maintainer, and must be elected by a supermajority of the steering committee. Likewise, maintainers can be removed by a supermajority of the steering committee or can resign by notifying one of the maintainers.
+New maintainers must already be contributors, must be nominated by an existing maintainer, and must be elected by a supermajority of the current team's maintainers. Likewise, maintainers can be removed by a supermajority of the current team's maintainers or steering committee or can resign by notifying one of the maintainers.
```
[the remark this answers is not in the document]
Comment (ForestEckhardt) [reply GOVERNANCE.md:48]: I think that is on a case by case basis and not necessarily the exact law of the land. My concern of adding `and` is that I believe the maintainers should be able to act autonomously of the steering committee. Also the usage of `and` would increase the votes needed for a super majority.

[the remark this answers is not in the document]
Comment (ForestEckhardt) [reply GOVERNANCE.md:48]: I can outline that case in some sort of exceptions heading. Maybe something to the effect of:
`In extenuating circumstances, i.e. long term absence of a maintainer, steering committee members may act as maintainers of any team in a voting capacity.`

Does that feel right or is it too narrow?

[the remark this answers is not in the document]
Comment (dmikusa-pivotal) [reply GOVERNANCE.md:48]: Sounds OK to me.

Do we also have/need an exception if there is an even number of maintainers and there is a tie? Would the tie breaker go to a steering committee member?

[the remark this answers is not in the document]
Comment (ForestEckhardt) [reply GOVERNANCE.md:48]: I used `i.e.` pretty intentionally. I think that I am more focused on the idea of an extenuating circumstance which could be any number of reasons.

[the remark this answers is not in the document]
Comment (ForestEckhardt) [reply GOVERNANCE.md]: I have given this section a bit of an update so feel free to give it a read and see if it lines up with what you would find appropriate.

**Pull request merged**